testbot_description/package.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<package>
+  <name>testbot_description</name>
+  <version>0.1.0</version>
+  <description>The testbot_description package</description>
+  <maintainer email="marta.lorenzini@iit.it">marta lorenzini</maintainer>
+
+  <license>BSD</license>
+  <url type="website">http://wiki.ros.org/testbot_description</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>urdf</build_depend>
+  <build_depend>message_generation</build_depend>
+  <build_depend>geometry_msgs</build_depend>
+  <build_depend>std_msgs</build_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>rospy</build_depend>
+  <build_depend>sensor_msgs</build_depend>
+  <build_depend>tf</build_depend>
+
+  <run_depend>urdf</run_depend>
+  <run_depend>joint_state_publisher</run_depend>
+  <run_depend>pr2_description</run_depend>
```

Comment:
Not a big problem, but I think we don't actually depend on `pr2_description`, so we can remove it.